### PR TITLE
Adding null check for tag input in DockerV2 - Porting to release

### PR DESCRIPTION
**Pull request by shigupt202**
**Task name**: DockerV2

**Description**: Porting fix to release from https://github.com/microsoft/azure-pipelines-tasks/pull/14265

**Documentation changes required:** (Y/N) Y

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/DockerV2/dockerbuild.ts [hunk 1]
```diff
@@ -51,7 +51,10 @@ export function run(connection: ContainerConnection, outputUpdate: (data: string
         imageNames.forEach(imageName => {
             if (tags && tags.length > 0) {
                 tags.forEach(tag => {
-                    tagArguments.push(imageName + ":" + tag);
+                    if(tag)
```
Comment (ajinkya599) [inline Tasks/DockerV2/dockerbuild.ts:54]: Do a `trim` before the null/empty check?

File: Tasks/DockerV2/task.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 2,
         "Minor": 181,
-        "Patch": 0
+        "Patch": 2
```
Comment (ajinkya599) [inline Tasks/DockerV2/task.json:17]: Why not Patch 1?

**Review by ajinkya599 — APPROVED** [review]

File: Tasks/DockerV2/dockerbuild.ts (hunk 1 above)
Comment (ajinkya599) [inline Tasks/DockerV2/dockerbuild.ts:54]: Isn't this change required in push as well?

File: Tasks/DockerV2/task.json (hunk 2 above)
Comment (shigupt202) [reply Tasks/DockerV2/task.json:17]: Hotfix branch has patch 1

File: Tasks/DockerV2/dockerbuild.ts (hunk 1 above)
Comment (shigupt202) [reply Tasks/DockerV2/dockerbuild.ts:54]: Discussed offline. This may not work for some cases.

Comment (shigupt202) [reply Tasks/DockerV2/dockerbuild.ts:54]: Added

**Pull request merged**